django/views/debug.py

```diff
@@ -86,7 +86,8 @@ def technical_500_response(request, exc_type, exc_value, tb, status_code=500):
     Create a technical server error response. The last three arguments are
     the values returned from sys.exc_info() and friends.
     """
-    reporter = ExceptionReporter(request, exc_type, exc_value, tb)
+    reporter_klass = get_default_exception_reporter()
+    reporter = reporter_klass(request, exc_type, exc_value, tb)
```

Comment:
`klass` is often used because `class` is a reserved keyword. I don't this it's necessary here. `reporter_class` would be fine.